### ISSUE TITLE
Impact, Prep, & Wishlist Grants Curation Residency

### ISSUE THREAD

**Issue by Ceramicwhite** [stacksgov/residence-program#35]
**What’s your name?**
Jasper Bower

**Best email on which to contact you?**
jasperbower@protonmail.com

**Personal Statement:**
The Grants program is an integral part of supporting the growth of Stacks in ways such as: supporting ambitious ideas; promoting open-source projects; and engaging with potential new developers. To accompany it, the Grant Ambassador initiative was a meaningful addition for prepping applicants and supporting current grantees. As the Ambassador program transitions, I understand many of the grantees that took advantage of the program will be surprised and disheartened by the recent change. The primary focus of my residency application is to initially focus on a smooth transition, while also striving to continue the core tenants of the Ambassador program over the next 12 months for both applicants and grantees. But rather than as a contractor of the Foundation, I am seeking the support and nomination from the community to continue these initiatives.
I followed Stacks development for a couple years and began to be involved in the community around the launch of mainnet. My interest in Stacks stemmed from a longtime belief in Bitcoin as the only timechain of importance. I worked as a member of the Foundation social team last year and more recently as a Grant Ambassador. 
As a Grant Ambassador, I am constantly meeting new developers and teams coming into Stacks excited to start building in the ecosystem but lacking an understanding of what the community wants and needs. This is where the curated wishlist grants have come into play and have had a huge impact, as a guide for those looking for ideas to build for Stacks. Despite the initial list never being publicly posted, almost half are under development in one way or another. I have offered ideas from the list to many teams that asked; they were always well received, and almost always one was chosen to be worked on. This is where the majority of my focus will be, on continued curation and proactively searching for individuals and teams to take on wishlist grants for the community. If anyone in the community has an idea for potential wishlist grants, please reach out and let's chat.

I’m grateful for any and all feedback from the community around my application for a full-time, 12 month residency.

**End Product:**

- Wishlist grant curation
  - Recruiting for and fulfilling the Wishlist grants.
   - Proactive Wishlists
    - Solicit Wishlist Ideas
   - Solicit Wishlist Applicants
  - Continually refining and expanding the list in conjunction with input and suggestions from the community.
  - Facilitate Wishlist Selection Process
 - Facilitating a 90 minute call (3) days a week that will be open to all people interested in applying for any type of a project (developer grant, event, community, education, resident, chapter) in partnership with Vane.
   - The goal of this standing call will be to help potential applicants better understand which type of project best suits their idea and how to craft a high impact application.
-   Applicant prep
  - Coaching solo & team applicants submit high-impact project ideas as a grant.
- Application Pre-Review
  - Feedback on projects and ideas.
- Onboarding
  - Introducing applicants/grantees to teams and suggesting potential cross-project collaboration.
- Continue work on stealth Foundation projects

**Breakdown:**

- 35% Applicant Prep + Onboarding + Application Pre-Review 
- 25% Impact Coaching
- 35% Wishlist Grant Program
- 5% Stealth projects

**Roadmap:**

3 months (Q3 2022):

- Continue supporting current grantees as they have come to expect and all around appreciate from the Grant Ambassador program.
- Supporting a smooth transition for grantees as the grants program is moved back to github.
- Update, seek out, & publish the first Wishlist of approved grants.
- On-going 3 days a week standing/office hour calls for 90 minutes.
- Seek out high quality founders.
- Launch of 1-2 stealth projects
- GH Posts on with updates on applications and on-going grants.

6 months (Q4 2022):

- Cont. Maintain wishlist grants list & recruit applicants
- Cont. supporting applicants &  current grantees
- Cont. On-going 3 days a week standing/office hour calls for 90 minutes.
- Cont. Seek out high quality founders. 
- Support and grow stealth projects.
- Cont. GH Posts with updates on applications and on-going grants.

9 months (Q1 2023):

- Cont. Maintain wishlist grants list & recruit applicants
- Cont. Supporting applicants & current grantees
- Cont. On-going 3 days a week standing/office hour calls for 90 minutes.
- Cont. Seek out high quality founders. 
- Cont. Maintain/support stealth projects.
- Cont. GH Posts with updates on applications and on-going grants.

12 months (Q2 2023):

- Cont. Maintain wishlist grants list & recruit applicants
- Cont. Supporting applicants & current grantees
- Cont. On-going 3 days a week standing/office hour calls for 90 minutes.
- Cont. Seek out high quality founders. 
- Cont. Maintain/support stealth projects (If necessary)
- Cont. GH Posts with updates on applications and on-going grants.

**Background & Skills:**

- A depth of knowledge of tools & apps deployed and in development for Stacks, Bitcoin, & Lightning
- A deep understanding of the Stacks community needs and sentiment.
- Extensive relationships with the Stacks community and the broader Bitcoin communities
- Ability to offer insights to teams on how to create high-impact projects for Stacks.
- Long term experience with the Bitcoin community and the technical side of Bitcoin
- A very technical understanding of Stacks, Bitcoin, and the relationship between them.

**Previous work in the Ecosystem:**

Foundation Social team (July-Feb)
- Created and managed the Foundation boombox giveaway campaigns. ([post](https://stacks.org/100k-stx-limited-edition-boomboxes-nft-giveaway))
- Developed and coordinated the Foundations Holiday giveaway which received over 35,000 addresses entered and 265 unique winners over 12 days.([post](https://www.reddit.com/r/stacks/comments/rhbe47/stacks_12_day_holiday_giveaway_20th_nye/))
- Created in partnership with Friedger & Boom the Surf Charity boombox campaign to support thesurferkids.com, raising enough Sats to fund the program for six months.([post](https://boom-wallet.medium.com/the-stacks-charity-cracker-244dd8c7ee09))
- Recommended and implemented the Discord revamp, from the adding of self-assigning roles in order to reduce the chaotic appearance of the discord to new users, to the formatting style now common across the communities Discords, and implementation of stricter rules and overhaul of CM abilities to combat scammers, and I’ll take credit for onboarding the CM legend Sleepi who has taken the Discord to new heights.
- Revamped & grew the Stacks subreddit by 110% during this time.([post](https://stacksfoundation.notion.site/Stacks-Social-Landscape-Growth-for-Q42021-7288691033144db6a980dab0f75d7058))
- Supported Social Team with the Twitter & Telegram channels with a focus on fact checking for inaccuracies about Stacks & Bitcoin.
- Continually recommended ditching “Can’t Be Evil” and suggested focusing on the Bitcoin narrative.

Grant Ambassador (April-Aug)
- Created an extensive Wishlist for grant ideas and surpassed Q2 KPI for in-progress wishlist grants either as a standalone grant, suggested as a milestone to applicants, or main deliverable of a residency even though the list never being made public.
- Successfully reached out to multiple teams and devs to pivot and build on Stacks.
- Coached applicants from denied applications to an approved potential high impact project.
- Participated in daily standing call/office hours that were well received and relied upon by applicants and grantees (Don't worry I'm still here for all of you, no matter what).

**Comment by dantrevino** [comment]
Anecdotal evidence leads me to believe that Stacks can do more/better outreach to new founders/developers looking for a way to build on Stacks.  I've personally heard stories during Purple Pill Friday Twitter space where developers are actively looking for a way to find out more information about grants, startup incubators, residency, etc.  I have personally advised several people on how to seek grants and/or residency.

I believe we need more of this type of work.

Feedback:
@Ceramicwhite I would like to see time spent on *existing* grantees as well.  If I missed that in your proposal, sorry.  I know @jennymith does a good job of following up on grantee needs already, but would like to see more of it and it is my gut feeling that she's already super busy.

**Comment by Ceramicwhite** [comment]
> Feedback: @Ceramicwhite I would like to see time spent on _existing_ grantees as well. If I missed that in your proposal, sorry. I know @jennymith does a good job of following up on grantee needs already, but would like to see more of it and it is my gut feeling that she's already super busy.

@dantrevino yes your right, that is a major part of the work we have been doing and I should have clearly stated that support will continue to be offered throughout their entire time as a grantee and beyond. 
Will amend my application, thanks for the feedback.

**Comment by obsidianbtc** [comment]
+1 on @Ceramicwhite as he is the most active participant inside the developer communities on Stacks. 

New and talented devs entering the space currently have little to no direction or support in getting up to speed or direction in contributing to the ecosystem. I feel strongly that to build a sustaining and healthy community; new developers must be shown wishlist items to jump-start their involvement in the space. 

The wishlist and grants program is currently the most transparent and helpful initiative at Stack's foundation for developers and founders. As an active member and founder in Stacks, I would like to see more energy allocated to supporting the ecosystem's greatest advocates, its founders, and devs.

I have great confidence in @Ceramicwhite as he has a deep technical understanding of both Bitcoin and Stacks and is deeply attuned to the ecosystem through his continual relationship-building and listening ear. 

TLDR: Listen to @Ceramicwhite

**Comment by JakeBlockchain** [comment]
+1 to this.

Speaking with Jasper, It's clear he has good grasp on the needs and opportunities  around the ecosystem. Especially the problems felt by one of the most important group, developers.
He has a rare ability to not only see where the puck is going, but his technical background allows him to give very tactical and actionable advice to developers thinking about building on Stacks.

I like this idea a lot.
One thing I would like to see is a weekly or bi-weekly video or blog post, something easily sharable and public facing, where we can get a pulse of what he is targeting and why.

**Comment by Hero-Gamer** [comment]
+1  I like Jasper's frank & practical approach to what is needed in the ecosystem. I can definitely see many areas I can collab Jasper with.

**Comment by Jamil** [comment]
+1 to this as well! Jasper was one of the first people to reach out to me when I first came into the Stacks ecosystem, and has been incredible in identifying and supporting developers. 

His work on growing the Stacks subreddit, organizing communities on Discord, proposing improvements, and advocating for founders and companies in the ecosystem is invaluable. I can't think of someone with as deep knowledge of the developer and founder ecosystem on Stacks, and with the technical knowledge to quickly and accurately evaluate the feasibility of companies and projects.

He's got the right direction and motivation to improve the Stacks ecosystem and is highly engaged with the community. For seeking out high-quality founders and supporting existing projects in the space, couldn't think of a better person than Jasper for the job.

**Comment by Alisammour** [comment]
Jasper has been invaluable when navigating the grants process as a contributing developer in the ecosystem. A rolling wishlist of initiatives to work on will also be helpful for new talent joining STX to address the growing pains of the network for scale.

He's always been incredibly receptive and diligent about improving processes or incorporating feedback from the grants weekly townhalls to help with creating a more supportive community environment. Looking forward to seeing this residency program scale out these efforts 10x!

**Comment by pseudozach** [comment]
+1 
I support Jasper in his current and future contributions to Stacks foundation and the ecosystem in general. He's technically capable and manages to identify gaps where most impact is possible. it's a joy to work with him as a developer to discuss bleeding edge innovation.
He helps devs identify what to focus their resources on which in my opinion is a superpower.
Looking forward to his curation where new devs coming into the ecosystem can immediately see projects they can build/contribute to and bounties/grants they can earn.

**Comment by jhammond2012** [comment]
+1 for me on this one too. Everyone has pretty much covered it above but this is a no-brainer here.

**Comment by jruffer** [comment]
If it were not for @Ceramicwhite I think we would have left this ecosystem a long time ago. He seems to have his finger on the pulse and knows how to get things done.  I would actually love to see this Residency have something with a sub-budget of X per month for him to complete these wishlist tasks without having to go through the grants process.  I feel there should already be enough trust between Stacks Jasper and the community to let him loose.  Release the Kraken!

**Comment by BowTiedSirJonathan** [comment]
No questions unanswered. No bit of help left ungiven. Jasper has always been there in times of need and I can truthfully vouch for him. Great guy!

**Comment by plutusdotbtc** [comment]
+1 For Jasper here. He has been an invaluable member of the Stacks community and the best person I could think of to represent the Stacks Foundation. He takes the time to get to know all of the builders in the ecosystem - everyone knows him because of this! Not only is he deeply involved in the Stacks ecosystem and community, but he has deep technical knowledge of bitcoin, giving him a perspective not many other people in the Stacks ecosystem have. He has proven to be an effective capital allocator for the foundation based on the grant proposals he has worked on. He thinks about what would actually benefit the Stacks ecosystem and what the technical needs are and allocates resources towards them. Because of his specific technical knowledge, he's able to sift out the projects that won't actually make an impact. As a builder in the ecosystem, I would be incredibly disappointed with the Stacks ecosystem if he wasn't chosen for this residency. He deserves this more than anyone I could think of.

**Comment by BowTiedDeployer** [comment]
+1
Staying in close touch with the devs and also having the technical skills to understand what every one of them is trying to build. 
All this while also being present in the discord chats and answering when needed. 
Combining the two, the needs of the community and the powers of the developers is truly a great skill and I can say Jasper is for sure the best fit for this position.

**Comment by RagnarLifthrasir** [comment]
+1 for Jasper's proposed end products and roadmap. I also strongly support Jasper in executing these in a residency.

Jasper navigated Jonathon and me through two grants. He patiently gave us detailed feedback and suggestions and responded promptly to my questions on Discord, text, and Twitter DMs. In addition, Jasper proactively spoke to me about an aspect of our grant that I considered and followed his advice. I appreciate how Jasper brings technical knowledge of Stacks and Bitcoin while also providing an awareness of community needs for product-market fit.

A grant residency that doesn't include Jasper would be akin to having a BLT sandwich without the bacon.

**Comment by Ceramicwhite** [comment]
Wow! Thank you for all the support and feedback means a lot


> One thing I would like to see is a weekly or bi-weekly video or blog post, something easily sharable and public facing, where we can get a pulse of what he is targeting and why.

@JakeBlockchain Agreed, this had been discussed as GH post in the grants repo. I'm not sure a weekly post would make sense right now but rather a post every other Friday may make more sense but yes I will definitely consider weekly and will post an update at the bare minimum twice a month.

**Comment by Albert-Present** [comment]
I have had the fortunate opportunity on several occasion to discuss various grant projects with Jasper.  I have experienced first hand that Jasper has a keen sense and eye for which projects are important for Stacks.  In addition, he is also talented in assessing the execution capability of grantees and how realistic a project scope is.  Although I can point out more strengths of Jasper such as enabling grantees in various ways, I think these should be reason enough for us to continue to support Jasper so that he may continue doing what he does best and grow the Stacks community together with us.

**Comment by wilsonbright** [comment]
+1 from me.  In the Stacks grant meeting in discord, Jasper was super helpful in clarifying the grant process and also sharing about wishlist grants.

**Comment by diopitis** [comment]
Jaspers wishlist grants have already filled a gap in the community for applications which need to exist for the future growth of Stacks. With his full time focus on this I can't even imagine all the developers he can bring into the ecosystem and the new projects that will come out of it!

**Comment by vidiabtc** [comment]
The most chill guy I know, Jasper never backs down from assisting others no matter what it may be. He's very easy-going and i can't imagine someone not discovering some sort of wisdom when talking to him! I genuinely think he'd knock this residency out of the park.

**Comment by ceramicwhite** [comment]
Despite the foundation's announcement to roll the ambassadors into the residence program so we could better serve the community, the foundation has decided to dismantle the ambassador program and declined my residency application.

It’s been an absolute pleasure to be a part of this amazing community. 
Thanks again everyone for the incredible support of my application.

**Comment by jennymith** [comment]
Hey Jasper, thanks for your cooperation throughout this process and to everyone in the community who participated as well. We're happy to provide more clarity on this decision below.

First of all, we want to start off by emphasizing two things:

1. This decision is not in disagreement with the sentiments expressed by the community. It’s clear from this thread that your work with Stacks has been and can continue to be valuable.
2. This decision does not suggest that you should discontinue your work with the Stacks community. It simply means that the work you proposed was not the right fit for this program.

Secondly, we acknowledge the shift from the Ambassador Program to the Residence Program and appreciate your patience in working with us through these unexpected changes. It became clearer, over several stages of comprehensive review, that the initial decision to transition ambassadors to residents did not make sense, both for the Residence Program and the future of the Grants Program. Understanding that you may not have planned for this situation, we will be disbursing a stipend to cover the time and effort you’ve spent on this application process. 

Regarding program fit: the Residence Program is open to all kinds of proposals, but ultimately, the objective of the program is to support novel research & development and experimentation. As both the [readme](https://github.com/stacksgov/residence-program) and the [announcement post](https://forum.stacks.org/t/exploring-a-stacks-residence-program) explain, this program “…caters to those looking to explore broader problems/unanswered questions [and is] suited to serving research needs, more ongoing governance-related projects, sustainable improvements to ecosystem infrastructure, and more.”

If you look at the current residents, you’ll see that the one thing they have in common is that they are all focusing on areas that are underexplored or underdeveloped, with the goal of building new tools or unlocking new insights and value for the community.  Given that you’ve already explored this model of grant reviews and support through the Ambassador Program, it wasn’t clear that you would land on any new insights through an extended residency. Also, as mentioned on the final call, there are already initiatives in place to address much of the work proposed here. 

Again, this is not to say that you shouldn’t continue to support the community in these ways! If you and others feel that there is a strong research & development case for this work, then we absolutely encourage you to resubmit an application next quarter or a clearly scoped grant with specific outcomes (these tend to be much easier to approve than residence positions). We will also be available for office hours next week to make sure that we capture and address all of your feedback moving forward.

**Comment by jennymith** [comment]
> Despite the foundation's announcement to roll the ambassadors into the residence program so we could better serve the community, the foundation has decided to dismantle the ambassador program and declined my residency application.
> 
> It’s been an absolute pleasure to be a part of this amazing community. Thanks again everyone for the incredible support of my application.

Hey Jasper, thanks for your cooperation throughout this process and to everyone in the community who participated as well. We're happy to provide more clarity on this decision below.

First of all, we want to start off by emphasizing two things:

1. This decision is not in disagreement with the sentiments expressed by the community. It’s clear from this thread that your work with Stacks has been and can continue to be valuable.
2. This decision does not suggest that you should discontinue your work with the Stacks community. It simply means that the work you proposed was not the right fit for this program.

Secondly, we acknowledge the shift from the Ambassador Program to the Residence Program and appreciate your patience in working with us through these unexpected changes. It became clearer, over several stages of comprehensive review, that the initial decision to transition ambassadors to residents did not make sense, both for the Residence Program and the future of the Grants Program. Understanding that you may not have planned for this situation, we will be disbursing a stipend to cover the time and effort you’ve spent on this application process. 

Regarding program fit: the Residence Program is open to all kinds of proposals, but ultimately, the objective of the program is to support novel research & development and experimentation. As both the [readme](https://github.com/stacksgov/residence-program) and the [announcement post](https://forum.stacks.org/t/exploring-a-stacks-residence-program) explain, this program “…caters to those looking to explore broader problems/unanswered questions [and is] suited to serving research needs, more ongoing governance-related projects, sustainable improvements to ecosystem infrastructure, and more.”

If you look at the current residents, you’ll see that the one thing they have in common is that they are all focusing on areas that are underexplored or underdeveloped, with the goal of building new tools or unlocking new insights and value for the community.  Given that you’ve already explored this model of grant reviews and support through the Ambassador Program, it wasn’t clear that you would land on any new insights through an extended residency. Also, as mentioned on the final call, there are already initiatives in place to address much of the work proposed here. 

Again, this is not to say that you shouldn’t continue to support the community in these ways! If you and others feel that there is a strong research & development case for this work, then we absolutely encourage you to resubmit an application next quarter or a clearly scoped grant with specific outcomes (these tend to be much easier to approve than residence positions). We will also be available for office hours next week to make sure that we capture and address all of your feedback moving forward.

**Comment by ceramicwhite** [comment]
>Given that you’ve already explored this model of grant reviews and support through the Ambassador Program, it wasn’t clear that you would land on any new insights through an extended residency. Also, as mentioned on the final call, there are already initiatives in place to address much of the work proposed here.

The application was based on suggestions and feedback by the foundation, prior to submission. During our most recent 1-on-1 call, you requested to have the community support initiatives removed and to focus entirely on the wishlist. With that being said, yesterday you stated, "The Foundation already has an initiative in place for Wishlist." This was your reasoning for the decision to decline my application which contradicts your instructions and feedback from the start.

Considering the Foundation proposed and coached all the Ambassadors on their residency applications, over a six week period, and then to only offer a two week stipend, is unethical. The entire process has not only been unsettling but also extremely disappointing towards those who have dedicated many hours to support the Stacks Foundation.

Wish you well on your future endeavors

**Comment by JakeBlockchain** [comment]
Disappointing to see how this is playing out. 

The focus of scope in this particular instance seems misguided to me. I understand why it's a focus but the work Jasper has been doing is largely formless and springs up organically from the developer needs that arise.
At a time when Stacks is getting some deserved shine, and more higher-caliber developers are taking interest in building here. You want people like Jasper to be available and on-call in some sense.

From my POV, these items are worth a grant by themselves and would be super helpful over the next year.
-Facilitating a 90 minute call (3) days a week that will be open to all people interested in applying for any type of a project (developer grant, event, community, education, resident, chapter)
-Application Pre-Review (Feedback on projects and ideas)
-Onboarding (Introducing applicants/grantees to teams and suggesting potential cross-project collaboration.
-Support current grantees all the way through to completion of their final Milestone.

This feels more like a residency or a full-time job position than a grant to me.

Jasper, Hope you find a way to keep doing the work you have and don't disappear in an ecosystem startup. 
But that's my only selfish desires haha.

**Comment by louiseivan** [comment]
@ceramicwhite was under my team when he was contracting at the Foundation. We spearheaded various efforts, and he was of great support—no doubt about his skill set. His technology understanding of protocols is top-notch.

@JakeBlockchain @RagnarLifthrasir 
I think everyone is mis-confusing residents here.  In simple terms, the residence program manages **programs**, not **people**. I want to emphasize the word **programs**.

Please refer to other ecosystems and how they fund various research and academic initiatives to build the ecosystem further. Ethereum is a great example, https://ethereum.org/en/community/grants/ we should be more output base rather than x hours committed. We should opt for something evergreen, which can be building blocks for the various verticals we are pushing. For example, if ETH pushes for SBT, research about its use cases is needed with technical and whitepaper documentation. 

Founders are on the end-user application layer and can be helped in various ways. There are tons of opportunities in the ecosystem to do so, such as founders lab, stacks ventures, and more. Foundation, in the long run, should focus on the core protocol. That's it.

**Comment by RagnarLifthrasir** [comment]
Hello @louiseivan. Regarding your explanation, I echo what @ceramicwhite said [in his reply](https://github.com/stacksgov/residence-program/issues/35#issuecomment-1213521824). And if the Foundation sees Jasper's value, it could have hired Jasper as an employee and given him a title such as "Grants Manager." The grants Jonathan and I received at Trajan has been a tremendous help, and I appreciate Will, Jasper, and the rest of the team for their hard work.

**Comment by RagnarLifthrasir** [comment]
I created a [GitHub issue with suggestions to improve the Foundation's governance](https://github.com/stacksgov/proposals/issues/14). I welcome feedback.
